src/cli.rs

```diff
@@ -175,6 +175,15 @@ pub enum Commands {
     #[clap(disable_version_flag = true)]
     #[clap(hide = true)]
     HelpMarkdown(HelpMarkdownArgs),
+
+    /// Clean up old packages from the vet cache
+    ///
+    /// Removes  packages which haven't been accessed in a while, and deletes
+    /// any extra files which aren't recognized by cargo-vet.
+    ///
+    /// In the future, many cargo-vet subcommands will implicitly do this.
+    #[clap(disable_version_flag = true)]
+    Gc(GcArgs),
```

Comment:
`#[clap(hide = true)]` is a thing you can use if you want to literally hide this from help messages, btw (I should figure out if we can get clap to group things so we can have all these debugging commands sectioned off clearly...)

Reply:
I wasn't sure that we necessarily wanted to hide it, I could see someone wanting to run it with a specific time or with --clean if they noticed the cargo-vet cache file getting a bit big.